lib/matplotlib/tests/test_sphinxext.py

```diff
@@ -10,10 +10,31 @@
 import pytest
 
 
-pytest.importorskip('sphinx',
-                    minversion=None if sys.version_info < (3, 10) else '4.1.3')
+def skip_if_no_sphinx(func):
+    pytest.importorskip(
+        'sphinx',
+        minversion=None if sys.version_info < (3, 10) else '4.1.3')
+    return func
```

Comment:
I don't understand what difference this makes. The decorators are all run when the functions are defined, which is at the top of the module, so this should happen on import, same as before.

Reply:
I think when I did this I thought there would be a reasonable way to test the changes without running sphinx (because it was all changes to text splitting etc), but ended up folding the new tests into the existing tests so it is moot.

I can just drop this commit.